src/assets/index.js

```diff
@@ -1 +1,2 @@
 export { default as landing_image } from "./images/landing_image.jpg";
+export { default as logo_image } from "./images/logo.png";
```

Comment:
This reference to `logo.png` should be `logo_image.png` based on the raw file.

Reply:
Resolved.